Cargo.toml

```diff
@@ -18,13 +18,12 @@ build = "build.rs"
 version_check = "^0.9.1"
 
 [dev-dependencies]
-wasm-bindgen-test = "0.2.50" # NOTE: keep in sync with wasm-bindgen version
+wasm-bindgen-test = "0.3.5" # NOTE: keep in sync with wasm-bindgen version
```

Comment:
remove `# NOTE: keep in sync with wasm-bindgen version` please, it looks that it isn't required anymore